### ISSUE TITLE
AI Upload board should not be in the Circuit Printer

### ISSUE THREAD

**Issue by CdrCross** [Skyrat-SS13/Skyrat13#2674]
I play a lot of AI. Every single round there is a science traitor, I get subverted, because it is stupidly easy to print an AI Upload board, build an Upload somewhere in maintenance, and then hack the AI. The Upload and Secure Storage are heavily secured areas, yet anyone who steps foot in Science can freely print a new Upload board. That should change.

**Comment by YakumoChen** [comment]
Should it though? That's just the game.

**Comment by SpookyTheFox** [comment]
I agree that the AI upload, as well as cyborg upload, and robotics control console all should not be printable. Contrary to popular belief, it's really not that much fun to be subverted. It's nothing but headache and risk for the silicon player. On the rare chance you get a traitor who knows how to manage silicon and has something fun, another silicon will often ruin it for the group. It is way too easy to print a board, grab some supplies to build a console, and carry it all around in a bluespace bag for instant, on the spot consoles that will never be found. Not only do antags do it, but also valid hunters who break character knowledge rules are often running around with essentially portable consoles.
Also, remember that if you are emagged as a borg, the robotics control console will round eliminate you, and that is it's only function on you. This necessitates dealing substantial damage to the station to prevent construction of more consoles, which often doesn't work anyway, simply in an attempt to stay in the round.

This is far from the only issue facing silicon players, but removing the boards from the printers is a good start.

**Comment by Yawet330** [comment]
^ Fuck being subverted. Fuck people who shittily spam subvert/reset silicons, fuck people who powergame subversion via building an AI upload at the engi circuit imprinter, before sprinting into their maint base to mass convert borgs/the AI for 'the lulz'

Robotics console is 50/50, a rework is planned from it, what I hear.

And yeah robotics console round eliminates you.

**Comment by YakumoChen** [comment]
The proper solution to hidden consoles is to do what TG does and automatically attach a GPS signal to them.

As for the boards, being able to print them is fine and completely removing them from being printable would be extremely unhealthy for game design and balance.

**Comment by CdrCross** [comment]
I don't think a GPS signal is much of a solution. An AI upload console stays online for less than ten seconds, before it exists as two boards in a backpack. There's also a huge dissonance between how difficult and secure it is to access the "Official" upload and upload boards, and how utterly trivial it is to make a new one.

**Comment by ShadeAware** [comment]
Removing upload boards will do nothing but allow antagonists to hoard all of the roundstart boards; and then by that point, instead of being able to actually fix the AI/Cyborgs, your only option would be to just kill the AI. This is not healthy in any respect, from both a roleplay and gameplay perspective.

**Comment by CdrCross** [comment]
> 
> 
> Removing upload boards will do nothing but allow antagonists to hoard all of the roundstart boards; and then by that point, instead of being able to actually fix the AI/Cyborgs, your only option would be to just kill the AI. This is not healthy in any respect, from both a roleplay and gameplay perspective.

The AI specifically has permission to bolt and protect secure tech storage from anyone but the Captain and CE under crewsimov, and if that board gets stolen, the capable Captain knows they need to card the AI before it gets subverted.

**Comment by YakumoChen** [comment]
Imagine being a pocket AI unable to so much as open a door for the entire round because someone stole one of two existing irreplaceable boards, without risk of subversion, and thinking that is a better game time than the core AI dynamic of law fuckery

**Comment by YakumoChen** [comment]
A single bolted door in a low traffic room is doing little to stop anyone either, and you will cry for boards to be printable again when the clever AI orders it's enginner borg to remove the only other existing law board from upload

**Comment by CdrCross** [comment]
Yakumo, I know for a fact that you rarely to never play silicons. Please stop projecting opinions.
Your first comment, regarding bein carded, is completely irrelevant in the context of this discussion, as if you've been carded, the RD can use the AI integrity restorer console to restore your laws on the card, without an upload. Your second comment is called foresight. If I and my borgs have made the effort to secure or dispose of the tools to reset our laws, that's a strategy. Unfortunately, that strategy matters basically naught when literally anyone can waltz into R&D, print two new board, and do whatever the hell they want with you.

**Comment by YakumoChen** [comment]
AI restorer does not touch laws. I have no idea where you got this info from but it is absolutely not true. Only an AI upload can change laws.

I am in fact an AI main, I just choose not to play Silicon on Skyrat (because rounds are long and high RP AI is boring and terrible and in fun)

**Comment by YakumoChen** [comment]
If i wanted to properly project an opinion it would be that the AI players complaining a traitor is using the dynamic of AIs to be an unreliable overseer that can be subverted and argue those same players are probably shit AIs who only play to validhunt and be Cameraman Beepsky instead of playing the Neutral Oversight that respects both traitors and crew, as the role intended.
